backend/onyx/seeding/load_yamls.py

```diff
@@ -127,6 +128,7 @@ def load_personas_from_yaml(
             display_priority=(
```

Comment:
what happens if both existing_persona.display_priority and persona.get("display_priority") are None. Is that OK?

Reply:
This should never happen